docs/spectral-rules/index.md

```diff
@@ -71,29 +78,31 @@ jobs:
       issues: read
       checks: write
       pull-requests: write
+      packages: read
 
     steps:
       - name: Checkout code
         uses: actions/checkout@v5
 
       - uses: actions/setup-node@v4
         with:
-          node-version: '22.x'
-          registry-url: 'https://npm.pkg.github.com'
+          node-version: "22.x"
+          registry-url: "https://npm.pkg.github.com"
           # Defaults to the user or organization that owns the workflow file
-          scope: '@ukhsa-collaboration'
+          scope: "@ukhsa-collaboration"
 
       - run: npm install @ukhsa-collaboration/spectral-rules
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install spectral
-        run: curl -L https://raw.github.com/stoplightio/spectral/master/scripts/install.sh | sh
-
-      - name: Lint example OpenAPI
+      - name: Lint OpenAPI specifications
         run: |
-          spectral --version
-          spectral lint "*.{json,yml,yaml}" -r ${{ GITHUB.WORKSPACE }}/node_modules/@ukhsa-collaboration/spectral-rules/.spectral.yaml -f github-actions
+          npx spectral --version
+          npx spectral lint "*.{json,yml,yaml}" -r ${{ GITHUB.WORKSPACE }}/node_modules/@ukhsa-collaboration/spectral-rules/.spectral.yaml -f github-actions
+
+          # Only lint OpenAPI files
+          # find . -name "openapi*.{yml,yaml,json}" -o -name "swagger*.{yml,yaml,json}" -o -name "*api*.{yml,yaml,json}" -not -path "./.git/*" -not -path "./node_modules/*" | \
```

Comment:
rather than providing a fairly complex bash example for this we could just mention that spectral uses [glob syntax](https://github.com/mrmlnc/fast-glob)

maybe we use this and rove the example and document glob a little?
```sh
npx spectral lint "@(openapi|swagger|*api)*.{json,yml,yaml}" -r ${{ GITHUB.WORKSPACE }}/node_modules/@ukhsa-collaboration/spectral-rules/.spectral.yaml -f github-actions
```

**the following glob `@(openapi|swagger|*api)*.{json,yml,yaml}` matches:**
>[!TIP]
> ```text
> openapi.json
> something.api.yaml
> swagger.json
> ```
**not**
> [!CAUTION]
> ```text
> /node_modules/openapi.yaml
> /.git/something.json
> ```